_tools/gulp/images/functions/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Don't ignore this file
+# This ensures that this folder
+# is tracked in version control
+# even if it is empty.
+!.gitignore
```

Comment:
This works fine, another method for this is adding an empty `.gitkeep` file instead of the `.gitignore`, then committing that, and only afterwards adding the folder ignore in the main `.gitignore`.  
https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/gitkeep-push-empty-folders-git-commit